pymc/distributions/logprob.py

```diff
@@ -126,6 +127,22 @@ def _check_no_rvs(logp_terms: Sequence[TensorVariable]):
         )
 
 
+def find_rng_nodes(
+    terms: Union[TensorVariable, Sequence[TensorVariable]], ops: Sequence[Type] = (RandomVariable,)
+):
+    """
+    Find all Variable instances that are outputs of `ops`
+    """
+    if not isinstance(terms, Sequence):
+        terms = [terms]
+    assert all(issubclass(cls, RandomVariable) for cls in ops)
+    return [
+        node
+        for node in aesara.graph.ancestors(terms)
+        if (node.owner and isinstance(node.owner.op, ops))
+    ]
+
+
```

Comment:
This does not seem necessary anymore? Even if so, it's place wouldn't be in this module.